js/main.js

```diff
@@ -21,6 +26,20 @@ function Main(){
   this.isStarted = false;
 }
 
+//Function for stopping and starting audio playback -Z
+function MusicSwitch(){
+if(sw){
+  audio.pause()
+  sw = false;
+}
+else if(sw === false){
+  audio.play();
+  sw = true;
+}
+}
+document.getElementById('mute').addEventListener('click', MusicSwitch);
+//restarts music once track has ended
+document.getElementById('sound').addEventListener('onended', audio.play());
```

Comment:
@zseiman you wont need this once you add the `loop` attribute to the `<audio>` tag.